behat-tests/README.txt

```diff
@@ -9,43 +9,44 @@ To set up your environment to run tests locally:
 1.  Download the Selenium stand-alone server JAR file:
     http://selenium-release.storage.googleapis.com/2.44/selenium-server-standalone-2.44.0.jar
 
-2.  Install Composer:
+2.  Selenium version 2.44 is only compatible with Firefox 33, but not 34 or above. You can download
```

Comment:
https://selenium.googlecode.com/git/java/CHANGELOG

"Updating Native events to support Firefox 24, 31, 32 and 33"

Selenium Standard Alone server change log specifies the compatibility is up-to and including Firefox 33.  I tried with 35 and 36, they both would not work. We are able to still use Firefox 34, and for the most part it may work, but we are risking known native events incompatibility that may bite us in the future. It can be very confusing and time consuming when that happens, especially on Jenkins.

There has been talks of resolving, but I could not find a timeline. In this case I recommend that we all use Selenium 2.44 .0 (starting yesterday, 2.45.0) and Firefox 33.1.1 across all dev and test environments.